watermarked/README.md

```diff
@@ -0,0 +1,18 @@
+# Tools and samples of Red Star OS watermarking
+
+Several samples of how Red Star's watermarking alters files are provided.
+
+## Watermark format
+
+The general format of the watermarks is:
+
+[24 byte watermark a]..[24 byte watermark n][4 byte checksum][3 byte EOF mark]
```

Comment:
I'd not use `a` to design the 1st watermark. I guess an interval `a ... n` is misleading, and recommend `1 ... n` instead.